client/settings/fraud-protection/index.js

```diff
@@ -0,0 +1,501 @@
+												<span className="component-modal__text--review">
+													{ __(
+														'authorized and held for review ',
+														'woocommerce-payments'
+													) }
+												</span>
+												{ __(
+													'if:',
+													'woocommerce-payments'
+												) }
+											</p>
+											<ul>
+												<li>
+													{ __(
+														"The card's issuing bank cannot verify the CVV.",
+														'woocommerce-payments'
+													) }
+												</li>
+												<li>
+													{ __(
+														'An order originates from an IP address outside your country.',
+														'woocommerce-payments'
+													) }
+												</li>
+												{ isDefaultCurrencyUSD ? (
+													<ExceedsDollarAmountRuleUSDStore level="standard" />
+												) : (
+													<ExceedsDollarAmountRuleNotUSDStore
+														level="standard"
+														defaultCurrency={
+															storeCurrency.name
+														}
+													/>
+												) }
+												<li>
+													{ __(
+														'The same card or IP address submits',
+														'woocommerce-payments'
+													) }{ ' ' }
+													<strong>
+														{ __( '5 orders' ) }
+													</strong>{ ' ' }
+													{ __( 'within' ) }{ ' ' }
+													<strong>
+														{ __( '72 hours.' ) }
+													</strong>
+												</li>
+											</ul>
+											<Button
+												className="component-modal__button--confirm"
+												onClick={ () =>
+													setStandardModalOpen(
+														false
+													)
+												}
+												isTertiary
+											>
+												{ __(
+													'Got it',
+													'woocommerce-payments'
+												) }
+											</Button>
+										</div>
+									</Modal>
+								) }
+							</div>
+							<p
+								className={
+									'fraud-protection__text--help-text'
+								}
+							>
+								{ __(
+									"Standard protection: Provides a standard level of filtering that's suitable for most businesses.",
+									'woocommerce-payments'
+								) }
+							</p>
+						</label>
+					</li>
+					<li>
+						<label htmlFor="fraud-protection__high-level">
+							<div className="fraud-protection-single-radio-wrapper">
+								<input
+									name={ 'fraud-protection-level-select' }
+									id={
+										'fraud-protection-level-select_high-level'
+									}
+									value={ 'high' }
+									type={ 'radio' }
+									checked={
+										'high' === currentProtectionLevel
+									}
+									onChange={ () =>
+										handleLevelChange( 'high' )
+									}
+								/>
+								<p className="fraud-protection-single-radio-wrapper__item">
+									{ __( 'High', 'woocommerce-payments' ) }
+								</p>
+								<HelpOutlineIcon
+									size={ 18 }
+									title="High level help icon"
+									onClick={ () => setHighModalOpen( true ) }
+								/>
+								{ isHighModalOpen && (
+									<Modal
```

Comment:
It would be nice if we could extract those modals from the main component.

Reply:
Agreed. I've added it to my to-do list.

Reply:
Modals have been extracted and placed in their own file in [b769432](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/b769432e4a042907866aa10735c9e01e8f3462ef). Let me know what you think of the new implementation.

Reply:
The changes are looking good! As a final nitpick, we can remove the fragment `<>` and return the short circuit directly:

```js
  return isHighModalOpen && (<Modal ...>);
```

or add some logic before and then return the component:
```js
  if ( ! isHighModalOpen ) {
    return null;
  }

  return <Modal ...>
```

The first solution is shorter but may backfire if the first operand `isHighModalOpen` doesn't return a boolean. You can easily get a `0` (it happens if you're doing something like `array.length && <>`) or an error because React can't render `undefined`. The idea is that it will return the value of the first operand if it's falsy, so `undefined && <Modal>` would return `undefined`.  This isn't the case with your code, but something to always keep in mind!